docs/irc_operators.md

```diff
@@ -0,0 +1,78 @@
+# IRC Operators
+
+This chapter describes useful information about the bridge for IRC operators or IRC users.
+
+The IRC bridge provides each Matrix user with one IRC connection in order to bridge them "natively"
+into the IRC network, as so they can largely be treated as real users. Due to the 1:1 connection system,
+it is often useful that the IRCD network provide the bridge host with a more relaxed ILINE limit depending
```

Comment:
I assume this is missing an "s". "the IRCD network" seems to be the subject here.
```suggestion
it is often useful that the IRCD network provides the bridge host with a more relaxed ILINE limit depending
```